newIDE/app/src/UI/Theme/Global/Animation.css

```diff
@@ -61,4 +61,9 @@
   to {
     transform: translateX(10px);
   }
+}
+
+@keyframes spin {
+  from {transform: translate(-50%, -50%) rotate(0);}
+  to   {transform: translate(-50%, -50%) rotate(360deg);}
```

Comment:
I guess the translate can be set somewhere else? I fear that reusing `spin` somewhere might have unexpected results